src/mono/cmake/configure.cmake

```diff
@@ -234,12 +234,9 @@ if(HOST_WIN32)
   # Dynamic lookup using ac_check_headers/ac_check_functions is extremly slow on Windows, espacially on msbuild.
   # Since majority of the checks above will fail on Windows host, we can just directly define the available static
   # API surface.
-  set(HAVE_ATEXIT 1)
```

Comment:
I think we need this and perhaps the others. Is that correct, @lateralusX ?

Reply:
These are unused. When we need them lets add them then.

Reply:
In dotnet/runtime perhaps, but they may be used in forks. If that's the case, I'd rather keep it.

Reply:
Can you point me to the fork you are referring to? The dead code is deleted in this repo. If fork is using it, it should be kept there.

Reply:
It's non-public, unfortunately. I want to make sure it's even used there first. If it's not, then my comment is moot.

Reply:
Not used so should be fine dropping and if needed in future we add it downstream.